nengo/ensemble.py

```diff
@@ -244,7 +244,7 @@ def ensemble(self):
     @property
     def probeable(self):
         """(tuple) Signals that can be probed in the neuron population."""
-        return ("output", "input") + self.ensemble.neuron_type.probeable
+        return self.ensemble.neuron_type.probeable + ("input",)
```

Comment:
Interesting effect of this is that the first element of the neuron type's `probeable` will be the default. Everything looks good for our neuron types, but I've added a comment to this docstring, just to help users (particularly those potentially implementing their own neuron types) to be aware. It'd be nice if we had a better place to note this for custom neuron types (e.g. if we had a notebook about custom neuron types), but I don't see a better place than here unfortunately.

Reply:
> the first element of the neuron type's `probeable` will be the default

Yes, but the first element of every neuron type's probeable is `"output"` so the behavior is the same and predictable: https://github.com/nengo/nengo/blob/master/nengo/neurons.py#L91

I don't believe it's possible to make a custom neuron type that doesn't have an `"output"` state, as it's automatically managed in the base class.

Reply:
Yeah, it would only be if somebody overrode that base class implementation and messed it up. For example, in `RatesToSpikesNeuronType`, we do such an override. On the one hand, it's really not our problem if somebody messes that up, but on the other hand, there's nowhere in the documentation that we say that the first probeable is the default (I didn't even realize that until looking at this PR), so to me it makes sense to mention it.

Reply:
I mentioned elsewhere, but this is documented in the `nengo.Probe` docstring, so I would document something specific to `NeuronType`s here, not something generic to probes.

Reply:
Having a comment in the Probe docstring is probably as good as having it here, so let's just leave it at that. I hadn't realized there was a comment there.